Userland/Applications/Browser/Tab.cpp

```diff
@@ -80,6 +80,30 @@ void Tab::view_source(const URL& url, const String& source)
     window->show();
 }
 
+void Tab::update_status(Optional<String> text_override, i32 count_waiting)
+{
+    if (text_override.has_value()) {
+        m_statusbar->set_text(*text_override);
+        return;
+    }
+
+    if (m_loaded) {
+        m_statusbar->set_text("");
+        return;
+    }
+
+    if (!m_navigating_url.has_value())
+        VERIFY_NOT_REACHED();
```

Comment:
Since it's no longer part of the branches, let's do this instead:

```suggestion
    VERIFY(m_navigating_url.has_value());
```

Reply:
Done :)